run-operator.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+
+export LICENSE=$(cat license.txt)
+
+if ! command -v minio &>/dev/null; then
+    brew install minio/stable/minio
+else
+    echo "MinIO is already installed."
+fi
+
+# Check if mc is installed
+if ! command -v mc &>/dev/null; then
+    brew install minio/stable/mc
+else
+    echo "mc (MinIO Client) is already installed."
+fi
+
+# MinIO configurations
+export MINIO_ROOT_USER=minio
+export MINIO_ROOT_PASSWORD=minio123
+export MINIO_ADDRESS="127.0.0.1:9000"
+export MINIO_BINARY=$(which minio)
+export MINIO_DATA_DIR="./minio-data"
+export MINIO_PID_FILE="/tmp/minio.pid"
+export MINIO_WANDB_BUCKET="wandb"
+export MINIO_ALIAS="myminio"
+
+# Check if MinIO is running
+is_minio_running() {
+    if [ -f "$MINIO_PID_FILE" ]; then
+        if ps -p $(cat "$MINIO_PID_FILE") >/dev/null 2>&1; then
+            return 0
+        else
+            rm "$MINIO_PID_FILE"
+            return 1
+        fi
+    else
+        return 1
+    fi
+}
```

Comment:
```
is_minio_running() {
    [ -f "$MINIO_PID_FILE" ] && ps -p $(cat "$MINIO_PID_FILE") >/dev/null 2>&1
}
```

we shouldn't be doing operators in a function that is checking

Reply:
Yah, this can be greatly improved, I was just concerned about another instances of minio potentially running on someones machine and was over engineering this. 😅